chapters/annotations.tex

```diff
@@ -1486,6 +1487,8 @@ \section{Graphical User Interface}\label{annotations-for-the-graphical-user-inte
 The \lstinline!showStartAttribute = true! is primarily intended for non-parameter values and avoids introducing a separate parameter for the \lstinline!start!-attribute of the variable.
 \end{nonnormative}
 
+If a non-parameter declaration has a modifier for the \lstinline!start!-attribute and does not have \lstinline!showStartAttribute = false! the \lstinline!start!- and \lstinline!fixed!-attributes may also be show.
```

Comment:
Apart from other concerns, there is a typo here:
```suggestion
If a non-parameter declaration has a modifier for the \lstinline!start!-attribute and does not have \lstinline!showStartAttribute = false!, the \lstinline!start!- and \lstinline!fixed!-attributes may also be shown.
```

Reply:
So what you're trying to say here could be summarized with:
> For a non-parameter with modifier for the `start`-attribute, it is recommended to treat absence of `showStartAttribute` the same way as `showStartAttribute = true`.

Reply:
Yes, it might be that clearer to just use that text.
The other changes for showStartAttribute are elsewhere, and not impacted by this.